### PR TITLE
[RFC][UN-7101] Create sign-in component on the ember addon

### PR DESCRIPTION
**Pull request by mmelo**
Refers to: https://uniplaces.atlassian.net/browse/UN-7101

Added sign in component

screen shot 2016-11-30 at 17 13 38

### CODE REVIEW

File: addon/templates/components/uni-signin.hbs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<div class="uni-signin__title">Log in</div>
+
+<form action="" class="uni-signin__step1">
```
Comment (montoias) [inline addon/templates/components/uni-signin.hbs]: no need for the class uni-signin__step1

**Review by montoias — CHANGES_REQUESTED** [review]

File: addon/templates/components/uni-signin.hbs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class="uni-signin__title">Log in</div>
```
Comment (AndreJoaquim) [inline addon/templates/components/uni-signin.hbs]: Remove hardcoded title string

File: addon/templates/components/uni-signin.hbs (hunk 1 above)
Comment (AndreJoaquim) [reply addon/templates/components/uni-signin.hbs]: Why this action?

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

**Comment by mmelo** [comment]
@AndreJoaquim @montoias both fixes made

**Pull request merged**